### ISSUE TITLE
floating bar chart

### ISSUE THREAD

**Issue by fugo** [sal0max/grav-plugin-shortcode-chart-js-plusplus#3]
I don’t get it to get the floating bar chart to work.
I found some info 
```
data: {
          labels: [1, 2, 3, 4, 5],
          datasets: [{
            label: 'data',
            data: [[-3, 5], [2, 10], [1, 3], [-4, -1], [4, 8]],
            backgroundColor: 'lightblue'
          }]
```
But how can I get this in the shortcode?

**Comment by sal0max** [comment]
There was a bug with multi dimensional arrays.
Should be fixed with f24cedddb01e3ef22919868c296aa7a1d7b73852.

This should work with the new version 1.0.7:

```
[chart
    type="bar"
    labels="[1, 2, 3, 4, 5]"]
    [dataset
        backgroundColor="lightblue"
        data="[[-3, 5], [2, 10], [1, 3], [-4, -1], [4, 8]]"
        label="data"]
[/chart]
```

Would love to hear from you if this worked.

**Comment by fugo** [comment]
Thanks 👍
I tought I'm too stupid for it 😆 tried a lot of different settings. Your plugin is great. 
Don't know charts.js before, I'm a 'Linux guy' working with graphviz and dot. 
Thank you for your fast respond
